assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -2186,5 +2188,94 @@ describe( 'core/modules modules', () => {
 				await waitForDefaultTimeouts();
 			} );
 		} );
+
+		describe( 'getDetailsLinkURL', () => {
+			it( 'should return null if module is not found', () => {
+				registry.dispatch( CORE_MODULES ).receiveGetModules( FIXTURES );
+
+				expect(
+					registry
+						.select( CORE_MODULES )
+						.getDetailsLinkURL( 'unregistered-module' )
+				).toBeNull();
+			} );
+
+			it( 'should return null if module does not define homepage', () => {
+				registry.dispatch( CORE_MODULES ).receiveGetModules( [
+					{
+						slug: 'search-console',
+						name: 'Search Console',
+						active: true,
+						connected: true,
+					},
+				] );
+
+				expect(
+					registry
+						.select( CORE_MODULES )
+						.getDetailsLinkURL( 'search-console' )
+				).toBeNull();
+			} );
+
+			it( 'should return module homepage', () => {
+				registry.dispatch( CORE_MODULES ).receiveGetModules( [
+					{
+						slug: 'search-console',
+						name: 'Search Console',
+						homepage: 'https://example.com',
+						active: true,
+						connected: true,
+					},
+				] );
+				registry
+					.dispatch( CORE_USER )
+					.receiveUserInfo( { email: 'test@example.com' } );
+
+				expect(
+					registry
+						.select( CORE_MODULES )
+						.getDetailsLinkURL( 'search-console' )
+				).toEqual(
```

Comment:
A very minor point, we should use `toBe()` in cases like these where we're performing a scalar comparison. Again, I'll apply the change myself to save the turnaround.

```suggestion
				).toBe(
```